best-practices/deploying_maintaining_aws.md

```diff
@@ -50,3 +48,28 @@ Verify changes and updates
 Apply changes and updates
 
 > terraform apply
+
+## Verify deployment
+
+### ECS
+
+When deploying new images to ECS, the log into the AWS console, and verify that the task enters a steady state. This can take several minutes.
```

Comment:
```suggestion
When deploying new images to ECS, log into the AWS console, and verify that the task enters a steady state. This can take several minutes.
```

Reply:
Good catch, fixed.